### PR TITLE
Atum

### PR DESCRIPTION
**Pull request by lucasccdias**
Hey, @Robinlovelace . Here is the first try with the new uptake model. Using this new approach we have almost 17k more bike trips, representing a 23% increase. Playing with the walking model now. If you have any comment or suggestion, please let me know!

### CODE REVIEW

**Comment by lucasccdias** [comment]
To do:
- [ ] focus on the top 25% (both for cycling and walking) -- maybe not a random sample using the weights, but selecting the top 25% really.
- [ ] convert walking trips with more than 5km to bike trips. 
- [ ] include ebikes scenario from `pct`.

**Comment by Robinlovelace** [comment]
I think we should test and merge this soon.

**Comment by lucasccdias** [comment]
@Robinlovelace Just made some small adjustments and things are looking good now. Let me know what you think.

File: .gitignore [hunk 1]
```diff
@@ -8,7 +8,7 @@
 *.geojson
 *.rar
 *.gpkg*
-*.csv
-*.png
```
Comment (Robinlovelace) [inline .gitignore:12]: 👍 we need the PNGs!

File: .gitignore [hunk 2]
```diff
@@ -8,7 +8,7 @@
...
 *.json
+*.png
```
Comment (Robinlovelace) [inline .gitignore:12]: I suggest removing this now.

**Review by Robinlovelace — APPROVED** [review]
How do the results look? I'm not sure based on a quick look at the code how it resolves the issue we were seeing with walking increasing excessively. My thinking was to calulate the new proportion of trips made by walking and then replace car trips accordingly, which may be in there now, would need to run the code to check but wanting to get feedback to you quickly @lucasccdias while working on other things. Great job by the way, this is clearly a big step forward so I approve the PR and we can always change details post merge.

**Comment by lucasccdias** [comment]
> How do the results look? I'm not sure based on a quick look at the code how it resolves the issue we were seeing with walking increasing excessively. My thinking was to calulate the new proportion of trips made by walking and then replace car trips accordingly, which may be in there now, would need to run the code to check but wanting to get feedback to you quickly @lucasccdias while working on other things. Great job by the way, this is clearly a big step forward so I approve the PR and we can always change details post merge.

Foot and bike trips are 1.27 and 2.35 times greater in the Curto-Prazo Scenario now (sounds reasonable?). There was a problem with this approach in routes with very high levels of foot trips and few car trips. The predicted increase in foot trips according with the `atum_foot` function was greater than the number of car trips in these routes -- and it generates the negative car trips we saw last week. In such cases, I consider a zero increase in foot trips in these routes.

**Pull request merged**